docs/guide/core-widgets.md

```diff
@@ -194,12 +221,15 @@ Several other Markdown shortcuts are enabled by default, including double tilde
 :::note
 Spaces in the syntax between the symbol and any text in the table are required, e.g., `## heading`. Equally, a lack of space between symbols and any text is also required for proper formatting, e.g., `**bold**`.
 :::
-### Configuring the `image` toolbar option
+### Configuring the `image` toolbar and insert menu option
+
+If you choose to enable the `image` toolbar option, which allows images to
+appear inline in text, you will usually want to also add it to the `insert` option so
+that the user can easily insert a brand new image without selecting tet first.
 
-If you choose to enable the `image` toolbar option, which is currently in beta and allows images to
-appear inline in text, you will usually want to also configure the `imageStyles` option
-to the `@apostrophecms/rich-text-widget` module in order to specify CSS classes the user is allowed to select
-for the image:
+In addition, you will likely want to configure the `imageStyles` option
+to the `@apostrophecms/rich-text-widget` module in order to specify CSS classes
```

Comment:
Hmmm, 'to' doesn't feel right to me here. It seems like 'of' would sound better. Up to you.